operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,502 @@
+ncn-w001   Ready    <none>          92d   v1.24.17   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,iscsi=sbps,katacontainers.io/kata-runtime=true,kubernetes.io/arch=amd64,kubernetes.io/hostname=ncn-w001,kubernetes.io/os=linux,node-discovery.cray.com/networks.hardware_management=true,node-discovery.cray.com/networks.node_management=true,node-discovery.cray.com/running=true,topology.kubernetes.io/zone=my-rack-x3000
+ncn-w002   Ready    <none>          92d   v1.24.17   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,iscsi=sbps,katacontainers.io/kata-runtime=true,kubernetes.io/arch=amd64,kubernetes.io/hostname=ncn-w002,kubernetes.io/os=linux,node-discovery.cray.com/networks.hardware_management=true,node-discovery.cray.com/networks.node_management=true,node-discovery.cray.com/running=true,topology.kubernetes.io/zone=my-rack-x3001
+ncn-w003   Ready    <none>          92d   v1.24.17   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,iscsi=sbps,katacontainers.io/kata-runtime=true,kubernetes.io/arch=amd64,kubernetes.io/hostname=ncn-w003,kubernetes.io/os=linux,node-discovery.cray.com/networks.hardware_management=true,node-discovery.cray.com/networks.node_management=true,node-discovery.cray.com/running=true,topology.kubernetes.io/zone=my-rack-x3002
+ncn-w004   Ready    <none>          92d   v1.24.17   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,iscsi=sbps,katacontainers.io/kata-runtime=true,kubernetes.io/arch=amd64,kubernetes.io/hostname=ncn-w004,kubernetes.io/os=linux,node-discovery.cray.com/networks.hardware_management=true,node-discovery.cray.com/networks.node_management=true,node-discovery.cray.com/running=true,topology.kubernetes.io/zone=my-rack-x3000
+ncn-w005   Ready    <none>          92d   v1.24.17   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,iscsi=sbps,katacontainers.io/kata-runtime=true,kubernetes.io/arch=amd64,kubernetes.io/hostname=ncn-w005,kubernetes.io/os=linux,node-discovery.cray.com/networks.hardware_management=true,node-discovery.cray.com/networks.node_management=true,node-discovery.cray.com/running=true,topology.kubernetes.io/zone=my-rack-x3001
+ncn-w006   Ready    <none>          91d   v1.24.17   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,iscsi=sbps,katacontainers.io/kata-runtime=true,kubernetes.io/arch=amd64,kubernetes.io/hostname=ncn-w006,kubernetes.io/os=linux,node-discovery.cray.com/networks.hardware_management=true,node-discovery.cray.com/networks.node_management=true,node-discovery.cray.com/running=true,topology.kubernetes.io/zone=my-rack-x3002
+```
+
+### 3 List Ceph zoning
+
+Check the `Ceph` zones created by executing the following command:
+
+   ```bash
+   ceph osd tree
+   ```
+
+   **Example Output**
+
+   The below sample output shows three racks configuration and their storage node distributions within the three racks.
+
+   ```text

```

Comment:
The formatting of this output does not look right -- the columns don't all line up. Is that how it shows up on the terminal?